protostar/commands/test/test_command.py

```diff
@@ -136,6 +144,7 @@ async def test(
         safe_collecting: bool = False,
         exit_first: bool = False,
         seed: Optional[int] = None,
+        report_slowest_tests_count: int = 0,
```

Comment:
We should generally avoid a first word that could be interpreted as a verb when naming variables:
```suggestion
        slowest_tests_to_report_count: int = 0,
```